files/es/web/javascript/reference/deprecated_and_obsolete_features/index.md

```diff
@@ -109,14 +109,8 @@ Estas características obsoletas se han eliminado por completo de JavaScript y y
 
 ### Objeto
 
-| Propiedad                                                                                | Descripción                                                                                      |
-| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
-| {{jsxref("Global_Objects/Object/count", "__count__")}}                 | Devuelve el número de propiedades enumerables directamente en un objeto definido por el usuario. |
-| {{jsxref("Global_Objects/Object/Parent", "__parent__")}}             | Apunta al contexto de un objeto.                                                                 |
-| {{jsxref("Global_Objects/Object/eval", "Object.prototype.eval()")}} | Evalúa una cadena de código JavaScript en el contexto del objeto especificado.                   |
-| {{jsxref("Object.observe()")}}                                                 | Observar de forma asincrónica los cambios en un objeto.                                          |
-| {{jsxref("Object.unobserve()")}}                                             | Elimina observadores.                                                                            |
-| {{jsxref("Object.getNotifier()")}}                                             | Crea un objeto que permite desencadenar sintéticamente un cambio.                                |
+- The [`Object.prototype.__proto__`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/proto) accessors are deprecated. Use [`Object.getPrototypeOf`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/getPrototypeOf) and [`Object.setPrototypeOf`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/setPrototypeOf) instead. This does not apply to the `__proto__` literal key in object literals.
+- The [`Object.prototype.__defineGetter__`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineGetter__), [`Object.prototype.__defineSetter__`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/__defineSetter__), [`Object.prototype.__lookupGetter__`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupGetter__), and [`Object.prototype.__lookupSetter__`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/__lookupSetter__) methods are deprecated. Use [`Object.getOwnPropertyDescriptor`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/getOwnPropertyDescriptor) and [`Object.defineProperty`](/es/docs/Web/JavaScript/Reference/Global_Objects/Object/defineProperty) instead.
```

Comment:
Is the same table as `en-US` just some outdated, maybe there was a confusion becasue there is two headers called "Object" in `en-US`

```suggestion
| Propiedad                                                                                | Descripción                                                                                      |
| ---------------------------------------------------------------------------------------- | ------------------------------------------------------------------------------------------------ |
| {{jsxref("Global_Objects/Object/count", "__count__")}}                 | Devuelve el número de propiedades enumerables directamente en un objeto definido por el usuario. |
| {{jsxref("Global_Objects/Object/Parent", "__parent__")}}             | Apunta al contexto de un objeto.                                                                 |
| {{jsxref("Global_Objects/Object/eval", "Object.prototype.eval()")}} | Evalúa una cadena de código JavaScript en el contexto del objeto especificado.                   |
| {{jsxref("Object.observe()")}}                                                 | Observar de forma asincrónica los cambios en un objeto.                                          |
| {{jsxref("Object.unobserve()")}}                                             | Elimina observadores.                                                                            |
| {{jsxref("Object.getNotifier()")}}                                             | Crea un objeto que permite desencadenar sintéticamente un cambio.                                |
```

Reply:
If you want to revert back to the table, the `__count__` and `__parent__` macros need to be changed to links like the the `__defineSetter__` ones, otherwise MD050 will break them when it lands